xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
The facility is injected as well? Huh, didn't know that. Formally we should validate it but it should be ok if we don't.